server/models/questions/updateQuestion.js

```diff
@@ -0,0 +1,87 @@
+const { pool } = require('../database')
+
+const updateTags = (tags, questionId) => {
+  return new Promise((resolve, reject) => {
+    let query = `INSERT INTO questions_tags(question_id, tag_id) VALUES`
+    const queryArr = []
+
+    tags.forEach((tag, idx) => {
+      query += `(?,?)`
+      if (idx !== tags.length - 1) {
+        query += `,`
+      }
+      queryArr.push(questionId, tag)
+    })
+
+    pool.query(query, queryArr, (error) => {
+      if (error) {
+        return reject(`Could not update tags: ${error}`)
+      }
```

Comment:
If the error is due to duplicate entry, then the error message can be sent accordingly.
Like 
```
if(error.code==='ER_DUP_ENTRY')
     return reject('The question already contains the requested tags')
```